docs/yul_language/functions.rst

```diff
@@ -0,0 +1,31 @@
+Functions
+=========
+
+Functions in Yul cannot access any variable outside their scope, i.e. they can
+only operate with the variables they receive as arguments. You can define types for arguments and returns. If no
+type is specified, the compiler will default to ``u256``.
```

Comment:
This is a little terse.

Reply:
I didn't want to make it prolix, but I'll improve it.